doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -817,7 +817,7 @@ end
 ## Garbage Collection Safety
 
 When passing data to a `@ccall`, it is best to avoid using the [`pointer`](@ref) function.
-Instead define a convert method and pass the variables directly to the `@ccall`. `@ccall`
+Instead define a [`cconvert`](@ref) method and pass the variables directly to the `@ccall`. `@ccall`
```

Comment:
Welcome and thanks for the contribution! It looks like CI is failing because it's not finding a documentation reference for `cconvert`. To fix that, you could do either
```suggestion
Instead define a [`cconvert`](@ref Base.cconvert) method and pass the variables directly to the `@ccall`. `@ccall`
```
or
```suggestion
Instead define a [`Base.cconvert`](@ref) method and pass the variables directly to the `@ccall`. `@ccall`
```
That said, I thought—though I could be mistaken—that the recommended approach was to define a `convert` method rather than `cconvert` since the former is more generally useful and the latter falls back to calling the former. In case case, you could do
```suggestion
Instead define a [`convert`](@ref) method and pass the variables directly to the `@ccall`. `@ccall`
```
which should work without qualification since `convert` is exported (`cconvert` is not).

Reply:
Mhh so here is my understanding:
As I understand the context is we have a julia object from which we need to obtain a pointer to be handled by C. But returning pointers from `Base.convert` is bad practice and  `Base.cconvert` is the better tool for this. Am I right?

Reply:
Yes, I think that's right. Rereading the context of this part of the documentation, it is specific to pointers, so perhaps the change to `cconvert` here is correct.